src/cdk/tree/nested-node.ts

```diff
@@ -60,6 +61,9 @@ export class CdkNestedTreeNode<T> extends CdkTreeNode<T> implements AfterContent
               protected _tree: CdkTree<T>,
               protected _differs: IterableDiffers) {
     super(_elementRef, _tree);
+    // The classes are directly added here instead of in the host property because classes on
+    // the host property are not inherited with View Engine.
+    this._elementRef.nativeElement.classList.add('cdk-tree-node', 'cdk-nested-tree-node');
```

Comment:
Passing in multiple classes to `add` won't work on IE11. We actually have a lint rule for it, but `_elementRef` probably resolves to `ElementRef<any>` so it wasn't caught. This should be split up into two calls.